server/index.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express')
+const app = express();
+const port = 8000;
+
+var dogs = [
+    {
+        name: 'pluto',
+        age: 5
+    },
+    {
+        name: 'scooby',
+        age: 1
+    }
+]
```

Comment:
Remove this and have a central JSON Message: 

```
[
   {
      "name":"All Notes"
   }
]
```

We can have more consistent Boilerplate Templates in my opinion 😄